assets/styl/_settings.styl

```diff
@@ -273,3 +273,37 @@ $svgLogoStyle ?= '
         fill rgba(255, 255, 255, .5)
 ' // @stylint ignore
 $svgDefaultMargins ?= 0 .3em -.1em 0
+
+// Z-Index
+//
+// Colette provides 3 default `z-index`
+//
+// ```
+// $zIndex ?= {
+//     base: 1
+//     overlay: 10
+//     layout: 20
+// }
+// ```
+//
+// **Usage example**
+// ```
+// /* stylus file */
+// z-index zIndex.layout + 4
+// ```
+//
+// will compile to
+// ```
+// /* CSS file */
+// z-index: 24
+//     /* styles */
+// ```
+//
+//
+//
+// Styleguide: Settings.Z-Index
+$zIndex ?= {
+    base: 1
+    overlay: 10
+    layout: 20
```

Comment:
Gap of 10 between each is enough ?

Reply:
I don't know, It's for collette, I think we will overide this object with gap of `100` in vega